packages/2d/src/utils/CanvasUtils.ts

```diff
@@ -70,6 +72,56 @@ export function drawRoundRect(
     rect,
   );
 
+  if (smoothCorners === true) {
```

Comment:
Since `smoothCorners` is a `boolean` this could just be `if (smoothCorners)`, no?

Reply:
Yes, it could be.